content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/__methods__/list_clusters.rb

```diff
@@ -0,0 +1,45 @@
+module ManageIQ
+  module Automate
+    module Infrastructure
+      module VM
+        module Transform
+          module Import
+            class ListClusters
+              def initialize(handle = $evm)
+                @handle = handle
+              end
+
+              def main
+                values_hash = {}
+                values_hash[''] = '-- select cluster from list --'
+
+                provider_id = @handle.root['dialog_provider']
+                if !provider_id.nil? && !provider_id.empty? && provider_id != '!'
+                  provider = @handle.vmdb(:ext_management_system, provider_id)
+                  if provider.nil?
+                    values_hash['!'] = '-- No clusters found --'
```

Comment:
@matobet 
We typically use nil => "None" as the default for 
https://github.com/ManageIQ/manageiq-content/blob/master/content/automate/ManageIQ/Cloud/Orchestration/Operations/Methods.class/__methods__/available_flavors.rb#L27